bugsnag-plugin-android-anr/src/main/jni/utils/string.c

```diff
@@ -0,0 +1,13 @@
+#include <string.h>
```

Comment:
This should instead reference the relative string header rather than the standard system directory. With the addition of another function in this file, or ordering changes, it could become a confusing bug later. [GCC docs on search paths](https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html)

```suggestion
#include "string.h"
```